chart/model_options.go

```diff
@@ -64,4 +64,11 @@ type Options struct {
 	Type string `json:"type"`
 	// Specifies the type of unit to use when displaying or labeling values     - Metric\\: Values represent decimal multiples. For example, if `unitPrefix` is Metric, 1K represents 1 kilobyte or 1000       bytes.     - Binary\\: Values represent binary multiples. For example, if `unitPrefix` is Binary, 1K represents 1 kibibyte or 1024       bytes.<br> **Note** This option is available for all values of `option.type` except `Text`
 	UnitPrefix string `json:"unitPrefix,omitempty"`
+	// The column configuration on logs chart
+	Columns []*Columns `json:"columns,omitempty"`
```

Comment:
Rather than referencing individual  types here I believe you should have define a `LogsChartOptions` and reference it here instead of `Columns`, `SortOptions` and `DefaultConnection`

Reply:
I have updated this to be nested, please take a look. I have one question here, these options appear on the chart object as follows

![Screen Shot 2022-07-22 at 11 28 52 AM](https://user-images.githubusercontent.com/61995858/180501585-a8313855-75eb-4838-8a36-79d3ad361c88.png)


Is it okay for the columns, sortOptions and defaultConnection to be nested  inside LogsListChartOptions?

Reply:
Sorry, you might be correct. I thought those fields are encapsulated on API level, looks I was wrong.

Reply:
no problem, I have reverted the change, please take another look and approve